app/routes/task_routes.py

```diff
@@ -1 +1,124 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from flask import request, abort, make_response
+from ..models.task import Task
+from .route_utilities import validate_model
+from ..db import db
+from sqlalchemy import asc, desc
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+bp = Blueprint("tasks_bp", __name__, url_prefix = "/tasks")
+
+@bp.post("")
+def create_new_task():
+    request_body = request.get_json()
+    if (
+        not request_body
+        or "title" not in request_body
+        or "description" not in request_body
+    ):
+        abort(make_response({"details": "Invalid data"}, 400))
+    new_task = Task.from_dict(request_body)
+    db.session.add(new_task)
+    db.session.commit()
+
+    return {
+            "task":{
+            "id" : new_task.id,
+            "title": new_task.title,
+            "description": new_task.description,
+            "is_complete": False
+            }
+        }, 201
```

Comment:
This repeats a lot of the functionality that your `create_model` helper function handles. 

How can you make use of that helper here to make this more concise and cut down on repeated code?